integr8ly/integreatly-operator
# MGDAPI-5859 - bump CRO version to 1.1.1

# Issue link
[MGDAPI-5859](https://issues.redhat.com/browse/MGDAPI-5859)

# What

Bump CRO version to 1.1.1

# Verification steps




- Install RHOAM on a CCS cluster.
- Confirm that a new version of CRO is present
- Verify upgrade is successful

quay.io/akramar/managed-api-service-index:1.37.0
quay.io/akramar/managed-api-service-index:1.38.0
